google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudBreakpointHandler.java

```diff
@@ -341,7 +341,7 @@ public void registerBreakpoint(
       BreakpointManager.getJavaBreakpoint(xIdeBreakpoint);
     if (!(cloudIdeBreakpoint instanceof CloudLineBreakpointType.CloudLineBreakpoint)) {
       LOG.error("breakpoint was not of the correct type to create on the cloud.  It was not a " +
-                "CloudLineBreakpoint");
+                "CloudLineBreakpoint"); //TODO akerekes: use class.getName() instead of hard coded name
```

Comment:
I think you can use your judgement on this one.  Also agree with @elharo that there's no need for a TODO, this is small enough that you should just do it, or not do it.